### ISSUE TITLE
Batch mode

### ISSUE THREAD

**Issue by adius** [alanshaw/markdown-pdf#47]
It should also be possible to specify several files in the cli and get a pdf for each file

**Comment by jasonb-vbt** [comment]
Sure, but that's easy enough to do on your own with a script or makefile. I'd prefer markdown-pdf to work on content-related issues (such as with links and images) first.

**Comment by alanshaw** [comment]
Yep, this would be good and should be relatively easy to implement given the [programmatic API already supports it](https://github.com/alanshaw/markdown-pdf#from-multiple-paths-to-multiple-paths).

PR would be gratefully accepted.
